components/NavigationDrawer/tests/snapshot/MDCBottomDrawerControllerSnapshotTests.m

```diff
@@ -152,4 +152,30 @@ - (void)testPresentedDrawerWithColoredViewsWithVerticalSizeClassCompact {
   [self generateSnapshotAndVerifyForView:self.bottomDrawerViewController.view];
 }
 
+- (void)testPresentedDrawerWithShadowElevation {
+  // Given
+  self.presentingViewController.view.frame = CGRectMake(0, 0, 375, 667);
+  self.containerViewController = [[FakeBottomDrawerContainerViewController alloc]
+      initWithOriginalPresentingViewController:self.presentingViewController
+                            trackingScrollView:nil];
+  self.containerViewController.contentViewController =
+      self.bottomDrawerViewController.contentViewController;
+  self.containerViewController.headerViewController =
+      self.bottomDrawerViewController.headerViewController;
+  self.containerViewController.contentViewController.view.backgroundColor = UIColor.whiteColor;
+  self.containerViewController.headerViewController.view.backgroundColor = UIColor.whiteColor;
+  self.containerViewController.drawerShadowColor = UIColor.blueColor;
+
+  // When
+  self.bottomDrawerViewController.view.bounds = CGRectMake(0, 0, 375, 667);
+  self.bottomDrawerViewController.contentViewController.preferredContentSize =
+      CGSizeMake(375, 1000);
+  self.bottomDrawerViewController.headerViewController.preferredContentSize = CGSizeMake(375, 80);
+  [self.bottomDrawerViewController.view addSubview:self.containerViewController.view];
```

Comment:
Don't you need to call `addChidViewController:` before adding the view as a subview? 

https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html#//apple_ref/doc/uid/TP40007457-CH11-SW12

Reply:
Done.